packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -1634,3 +1634,159 @@ test.describe('binding_property_non_reactive warn', () => {
 		expect(is_warning_thrown).toBeFalsy();
 	});
 });
+
+test.describe('navigation and redirects should be consistent between web native and sveltekit based', () => {
+	function expectPath(page) {
+		return expect(new URL(page.url()).pathname);
+	}
+
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/navigation/teststart');
+		expectPath(page).toBe('/navigation/teststart');
+	});
+
+	test.describe('navigating outside the app on sameorigin', () => {
```

Comment:
Here I compare browser history between plain web navigation and sveltekit's goto navigation. I claim that browser history should be the same no matter which type of navigation I perform